tenable/io/v3/vm/agents/agents_iterator.py

```diff
@@ -0,0 +1,27 @@
+'''
+Agents Iterator
+'''
+from tenable.io.base import TIOIterator
+
+
+class AgentsIterator(TIOIterator):
+    '''
+    The agents iterator provides a scalable way to work through agent result
+    sets of any size.  The iterator will walk through each page of data,
+    returning one record at a time.  If it reaches the end of a page of
+    records, then it will request the next page of information and then
+    continue to return records from the next page (and the next, and the next)
+    until the counter reaches the total number of records that the API has
+    reported.
+
+    Attributes:
+        count (int): The current number of records that have been returned
+        page (list):
+            The current page of data being walked through.  pages will be
+            cycled through as the iterator requests more information from the
+            API.
+        page_count (int): The number of record returned from the current page.
+        total (int):
+            The total number of records that exist for the current request.
+    '''
+    pass
```

Comment:
Can you please update here as NotImplemented

Reply:
Added.